CHANGELOG

```diff
@@ -1,3 +1,6 @@
+* Mon Mar 25 2019 Joseph Sharkey <shark.bruhaha@gmail.com> - 6.2.0-0
+- Standardized cron datatypes to use the Simplib::Cron::### types
```

Comment:
Why is this a minor version bump? I would think this is a patch bump. It is completely backward compatible and simply allows more flexibility in values allowed for cron-related parameters.

Reply:
It was new functionality for those fields so I wasn't sure what the cutoff for a patch bump was.

Reply:
It is not backwards compatible because the new datatype allows for Integers, not just Strings.